src/pl/michaeli/service/ConfirmationResource.java

```diff
@@ -0,0 +1,220 @@
+package pl.michaeli.service;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.GenericEntity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import pl.michaeli.model.BookedSeat;
+import pl.michaeli.model.ErrorMessage;
+import pl.michaeli.model.Reservation;
+import pl.michaeli.model.Seat;
+import pl.michaeli.spring.config.AppConfig;
+import pl.michaeli.spring.dao.BookedSeatsDAO;
+import pl.michaeli.spring.dao.ReservationsDAO;
+import pl.michaeli.spring.dao.ScreeningsDAO;
+import pl.michaeli.spring.dao.SeatsDAO;
+import pl.michaeli.spring.dao.TicketTypesDAO;
+
+
+@Path("confirm")
+public class ConfirmationResource {
+
+	@POST
+	@Consumes({MediaType.APPLICATION_XML,MediaType.APPLICATION_JSON})
+    public Response confirmReservation(Reservation reservation) {
+
+		String name = reservation.getName();
+		String surname = reservation.getSurname();
+		int screeningId = reservation.getScreeningId();
+		List<BookedSeat> seatsToBook = reservation.getBookedSeats();
```

Comment:
Trochę konfundujące nazwy. Funkcja nazywa się getBookedSeats, a zmienna seatsToBook. to jak w końcu, booked czy to book?

Reply:
Jakoś mi się wydawało, że taka nazwa będzie bardziej czytelna - żeby podkreślić, że jeszcze musi pójść walidacja. Ale może i racja..